docs-src/generate.js

```diff
@@ -83,13 +89,14 @@ function genStyles(cb) {
       path.join(__dirname, '../scss'),
     ],
     sourceMap: true,
-  }, cb || function(err, {css}) {
+  }, cb || function(err, results) {
     if(err) {
```

Comment:
Also here: we were trying to use the wrong name for the err var, and it kept killing the dev server instead of logging. Also: avoiding access to {css} in case of parse errors, because `results` is null in that case.